resources/install.rb

```diff
@@ -191,7 +215,7 @@
       end
       unless cvlt_registered?
         puts "CommVault registration unsuccesful after #{new_resource.registration_timeout} seconds, uninstalling"
-        # Call uninstall
+        # Call uninstall - TODO
```

Comment:
```suggestion
        # TODO: Call uninstall
```

This is a common way to show things to be done, some editors even pick it up and show different highlighting.

Reply:
Done